recipes/base_install.rb

```diff
@@ -122,6 +122,14 @@
   mode '0744'
 end
 
+cookbook_file 'setup-ephemeral.service' do
+  path '/etc/systemd/system/setup-ephemeral.service'
+  owner 'root'
+  group 'root'
+  mode '0644'
+  only_if { node['init_package'] == 'systemd' }
```

Comment:
question: why we don't consider init.d here?

Reply:
because we don't support anymore OSs that use `init.d`